easybuild/tools/testing.py

```diff
@@ -265,8 +323,15 @@ def create_test_report(msg, ecs_with_res, init_session_state, pr_nrs=None, gist_
     for key in sorted(environ_dump.keys()):
         if env_filter is not None and env_filter.search(key):
             continue
-        else:
-            environment += ["%s = %s" % (key, environ_dump[key])]
+        value = environ_dump[key]
+        if any(re.match(rgx, value) for rgx in DEFAULT_EXCLUDE_FROM_REPORT_RGX):
+            continue
+        environment += ["%s = %s" % (key, value)]
+
+    environment = list(filter(
+        lambda x: not any(y in x.upper() for y in DEFAULT_EXCLUDE_FROM_REPORT + _exclude_env_from_report),
```

Comment:
Should the values in `DEFAULT_EXCLUDE_FROM_REPORT` be globbed (not just `SECRET`, but `*SECRET*`?)

Reply:
I am not using regex there, we are just asking if the name of the variable contains any of the values in `DEFAULT_EXCLUDE_FROM_REPORT`